src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -1881,13 +1881,22 @@ protected override void ProcessRecord()
 
         internal override IList<PSModuleInfo> ImportModulesUsingWinCompat(IEnumerable<string> moduleNames, IEnumerable<ModuleSpecification> moduleFullyQualifiedNames, ImportModuleOptions importModuleOptions)
         {
+            IList<PSModuleInfo> moduleProxyList = new List<PSModuleInfo>();
+#if !UNIX
+            var winPSVersionString = Utils.GetWindowsPowerShellVersionFromRegistry();
+            if (!winPSVersionString.StartsWith("5.1", StringComparison.OrdinalIgnoreCase))
```

Comment:
Make sense pass this with Force parameter?

Reply:
I see where it might be useful; however that would be a bad user experience in case of failure - we got this feedback internally when someone hit this - this actually was the reason why this PR was created.